src/monosynth.js

```diff
@@ -0,0 +1,358 @@
+'use strict';
+define(function (require) {
+
+  var p5sound = require('master');
+  var AudioVoice = require('audioVoice');
+  require('sndcore');
+  require('oscillator');
+  require('env');
+  require('filter');
+
+  /**
+    *  An MonoSynth is used as a single voice for sound synthesis.
+    *  This is a class to be used in conjonction with the PolySynth
+    *  class. Custom synthetisers should be built inheriting from
+    *  this class.
+    *
+    *  @class p5.MonoSynth
+    *  @constructor
+    *  @example
+    *  <div><code>
+    *  var monosynth;
+    *  var x;
+    *  
+    *  function setup() {
+    *    monosynth = new p5.MonoSynth();
+    *    monosynth.loadPreset('simpleBass');
+    *    monosynth.play(45,1,x=0,1);
+    *    monosynth.play(49,1,x+=1,0.25);
+    *    monosynth.play(50,1,x+=0.25,0.25);
+    *    monosynth.play(49,1,x+=0.5,0.25);
+    *    monosynth.play(50,1,x+=0.25,0.25);
+    *  }
+    *  </code></div>
+    **/
+
+  p5.MonoSynth = function () {
+    AudioVoice.call(this);
+
+    this.oscillator = new p5.Oscillator();
+    this.oscillator.disconnect();
+
+    this.env = new p5.Env();
+    this.env.setRange(1, 0);
+    this.env.setExp(true);
+
+    //set params
+    this.setADSR(0.02, 0.25, 0.05, 0.35);
+
+    // filter
+    this.filter = new p5.Filter('highpass');
+    this.filter.set(5, 1);
+
+    // oscillator --> env --> filter --> this.output (gain) --> p5.soundOut
+
+    this.oscillator.connect(this.filter);
+    this.env.setInput(this.oscillator);
```

Comment:
I'm not sure if the chain is working (testing it out, all the examples are silent)

Maybe we just need `p5.MonoSynth.prototype = Object.create(p5.AudioVoice.prototype);`